src/components/TodoHeader.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div class="flex justify-between">
-    <h1 class="text-7xl font-bold">To do List</h1>
+  <div class="flex items-center justify-between">
+    <h1 class="text-4xl font-bold sm:text-7xl">To do List</h1>
     <PlusCircleIcon
       class="w -16 h-16 cursor-pointer text-green-400"
```

Comment:
Be careful, don't add extra spaces to the tailwind classes because it won't work :)